amd_build/build_pytorch_amd.py

```diff
@@ -0,0 +1,47 @@
+import shutil
+import subprocess
+import os
+
+cwd = os.getcwd()
+proj_dir = os.path.dirname(cwd)
+out_dir = os.path.join(os.path.dirname(proj_dir), "pytorch_amd")
```

Comment:
I don't see updates to `.gitignore`. Is the copied source tree ignored appropriately by git?